jigsaw-game/client/jigsaw.py

```diff
@@ -0,0 +1,102 @@
+import client
+from pygame import Rect, Surface
+
+TOP = BOTTOM = 1
+LEFT = RIGHT = 0
+
+
+class JigSaw(Surface):
+    """This is the Surface that the user interacts with the puzzle"""
+
+    def __init__(self, screen: Surface, size: tuple[int, int]):
+        """Set Jigsaw position"""
+        super(JigSaw, self).__init__(size)
+        self.screen = screen
+        self.size = size
+        self.set_bounds()
+
+    def set_bounds(self):
+        """Called to set the real surface coords"""
+        screen_bounds = self.screen.get_rect()
+        surface_bounds = self.get_rect()
+        x_offset = (screen_bounds.width - surface_bounds.width) // 2
+        y_offset = (screen_bounds.height - surface_bounds.height) // 2
+        self.rect = Rect(
+            x_offset, y_offset, self.size[0] + x_offset, self.size[1] + y_offset
+        )
+        self.center = (x_offset, y_offset)
+
+    def translate(self, position: tuple[int, int]):
```

Comment:
```suggestion
    def translate(self, position: tuple[int, int]) -> None:
```
I think this and a few others are already covered in other PRs but not sure.

Reply:
No this is supposed to return tuple of int's

Reply:
You are correct. An oversight on my part.